backend/src/exercise/patient-ticking.ts

```diff
@@ -13,7 +12,7 @@ import {
 /**
  * The count of assigned personnel and material that cater for a {@link Patient}.
  */
-type Catering = { [key in PersonnelType | 'material']: number };
+type Catering = { [key in string | 'material']: number };
```

Comment:
can just be key in string

Reply:
I also thought about suggesting this change, but I think that it actually does make sense to explicitly specify `material` to express that "the key is either some string (a personnel type) or literal `'material'`"

Reply:
Then we should do it properly and require it, I think:

```suggestion
type Catering = { [key in string]: number } & { material: number };
```